crates/relayer-cli/src/cli_utils.rs

```diff
@@ -102,3 +102,88 @@ pub fn spawn_chain_counterparty<Chain: ChainHandle>(
         channel_connection_client,
     ))
 }
+
+/// Check that the relayer can send on the given channel and ensure that channels and chain identifiers match.
+/// To do this, fetch from the source chain the channel end, then the associated connection
+/// end, and then the underlying client state; finally, check that this client is verifying
+/// headers for the destination chain.
+pub fn check_can_send_on_channel<Chain: ChainHandle>(
+    src_chain: &Chain,
+    src_channel_id: &ChannelId,
+    src_port_id: &PortId,
+    dst_chain_id: &ChainId,
+) -> Result<(), eyre::Report> {
+    use eyre::eyre;
+    use tracing::debug;
+
+    use ibc_relayer::chain::requests::{
+        IncludeProof, QueryChannelRequest, QueryClientStateRequest, QueryConnectionRequest,
+        QueryHeight,
+    };
```

Comment:
It seems a bit non-standard to import these types at the function level. Is there any particular reason for doing it this way? If not, perhaps we should remain consistent and import these at the top of the file.

Reply:
[21de7d4](https://github.com/informalsystems/hermes/pull/2682/commits/21de7d431fcf574c42d50f018fc1a19e4d439cea)